src/upstreams.go

```diff
@@ -1,9 +1,10 @@
 package main
 
 import (
+	"cmp"
```

Comment:
cmp package was introduced in go v1.21, but the last version that support Windows 7 (The NT6.1 build) was Go 1.20. Hence, this cannot be used in Zoraxy v3 (maybe you need to wait until Zoraxy v4 after we dropped windows 7 support before cmp package can be used)

Reply:
I've changed back to manual compare function